### DISCUSSION TITLE
Benefits for standard RDS PostgreSQL

### DISCUSSION THREAD

**Discussion by neugeeug** [aws/aws-advanced-jdbc-wrapper#216] [General]
Hi, 

We have a few Java microservices working with PostgreSQL RDS with Multi-AZ enabled.
Is there any advantage of using your driver for that scenario as well or it is mainly dedicated for Aurora ?

Br.
Eugeniusz

**Comment by davecramer** [comment]
Hi,

Unfortunately at the moment there is no failover enhancement with RDS with Multi-AZ. However you could still take advantage of the IAM authentication and Secrets manager
